pom.xml

```diff
@@ -24,7 +24,7 @@
 
         <!-- TODO: Bump version (some breaking API changes). -->
         <!--<common.version>2.2020.10.15_11.43-b1f02e7bd6ae</common.version>-->
-        <common.version>2.2020.08.07_16.18-e6be6d6a09e6</common.version>
+        <common.version>2.2021.02.03_08.08-9d0c8281f06b</common.version>
```

Comment:
Denne versjonen av common har tatt bort spesiell håndtering av serialisering og deserialisering av `java.util.Date`. Bør enten unngå bruk av `java.util.Date`, eller ta inn igjen denne håndteringen i common før denne PR'en merges.

Reply:
Støtten for `java.util.Date` er lagt tilbake i siste versjon av common.